grid/recruitment/update-candidate/profile.supr

```diff
@@ -0,0 +1,397 @@
+"""
```

Comment:
How is this different to #335  grid/recruitment/update-candidate/profile.supr ?

Reply:
Sorry I forgot to change the base branch.